skype/skype-ps/skype/Get-CsUserSettingsPageConfiguration.md

```diff
@@ -0,0 +1,104 @@
+---
+external help file: Microsoft.Rtc.Management.dll-Help.xml
+online version: https://docs.microsoft.com/powershell/module/skype/get-csusersettingspageconfiguration
+applicable: Skype for Business Online
+title: Get-CsUserSettingsPageConfiguration
+schema: 2.0.0
+manager: bulenteg
+author: tomkau
+ms.author: tomkau
+ms.reviewer: rogupta
+---
+
+# Get-CsUserSettingsPageConfiguration
+
+## SYNOPSIS
+This command is used to get the current URI for setting PSTN calling and voice mail related settings.
+
+## SYNTAX
+
+### Identity (Default)
+```
+Get-CsUserSettingsPageConfiguration [[-Identity] <XdsIdentity>] [-LocalStore] [<CommonParameters>]
+```
+
+### Filter
+```
+Get-CsUserSettingsPageConfiguration [-Filter <String>] [-LocalStore] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This command is used to get the current URI for setting PSTN calling and voice mail related settings. The Skype for Business Administrator may choose to update these URI based on their configured PSTN calling and Voice mail services using Set-CsUserSettingsPageConfiguration:
+- PstnCallingUri: The URI of the PstnCalling User settings page
+- VoicemailUri: The URI of the Voicemail tab in User Settings Portal
+
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsUserSettingsPageConfiguration
+```
+
+This example is used to retrieve the Global User Settings Page Configuration.
+
+## PARAMETERS
+
+### -Filter
+The Filter parameter allows you to limit the number of results based on filters you specify.
+
+```yaml
+Type: String
+Parameter Sets: Filter
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+The Identity parameter is a unique identifier that designates the scope.
+
+```yaml
+Type: XdsIdentity
+Parameter Sets: Identity
+Aliases:
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -LocalStore
+Retrieves the user pool information from the local replica of the Central Management store rather than from the Central Management store itself.
```

Comment:
```suggestion
This parameter is reserved for internal Microsoft use.
```

Reply:
@get-itips what do you think?

Reply:
my bad, @dariomws suggested correctly to put that text based on the metadata that said this was an Online cmdlet, however it is a Server cmdlet, pushed another commit to correct it.

Reply:
Thanks!